Source/Scene/ModelComponents.js

```diff
@@ -647,21 +647,21 @@ function Components() {
 }
 
 /**
- * A texture.
+ * The components for reading a texture.
```

Comment:
I guess this answers my previous comment - all documentation about a `TextureReader` object just use this description.